.github/workflows/release.yml

```diff
@@ -6,58 +6,23 @@ on:
       - '**'
 
 jobs:
-  create_pre_release:
-    name: Create pre-release
-
-    runs-on: ubuntu-latest
-
-    steps:
-      # The pre-release must be created only once, hence the split
-      # into multiple jobs with different `strategy`.
-      - name: Create a Github pre-release
-        id: create_pre_release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          tag_name: ${{ github.ref }}
-          release_name: ${{ github.ref }}
-          draft: false
-          prerelease: true
-
-      - name: Output `release_url` into a temporary file
-        run: echo "${{ steps.create_pre_release.outputs.upload_url }}" > release_url.txt
-
-      - name: Save the `release_url` temporary file
-        uses: actions/upload-artifact@v1
-        with:
-          name: release_url
-          path: release_url.txt
-
-  release:
-    name: Build and publish wheels
-
-    needs: [create_pre_release]
+  build:
+    name: Build
 
     strategy:
       matrix:
-        # The job runs on different OS.
-        target: [
-          # Linux, amd64
-          { id: 'linux-amd64', os: 'ubuntu-latest', rust-target: 'x86_64-unknown-linux-gnu', rust-toolchain: 'stable' },
-          # macOS, amd64
-          { id: 'darwin-amd64', os: 'macos-latest', rust-target: 'x86_64-apple-darwin', rust-toolchain: 'stable' },
-          # Windows, amd64
-          { id: 'windows-amd64', os: 'windows-latest', rust-target: 'x86_64-pc-windows-msvc', rust-toolchain: 'stable' },
-          # Windows, x86
-          #{ id: 'windows-x86', os: 'windows-latest', rust-target: 'i686-pc-windows-msvc' },
-          # Linux, aarch64
-          #{ id: 'linux-aarch64', os: ['self-hosted', 'aarch64'], rust-target: 'aarch64-unknown-linux-gnu', rust-toolchain: 'stable' },
```

Comment:
Why Windows has been removed?

Reply:
It has been removed from both workflows. I want to have these validated and then work on making Windows work.